tools/rdiff-backup.spec.template

```diff
@@ -53,9 +56,13 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.so
 %{_datadir}/bash-completion/completions/rdiff-backup
 %ghost %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.pyo
-%doc CHANGELOG COPYING docs/FAQ.md docs/examples.md docs/README.md docs/DEVELOP.md
+%doc CHANGELOG COPYING README.md
+%doc docs/FAQ.md docs/examples.md docs/DEVELOP.md docs/Windows-README.md
```

Comment:
stupid question but wouldn't `%doc docs/*.md` work and be easier to maintain?

Reply:
Yeah, probably.  I'll update them with that.

Reply:
Done.